### DISCUSSION TITLE
Add `rg-jump` command inspired by git-jump

### DISCUSSION THREAD

**Discussion by phcerdan** [BurntSushi/ripgrep#2543] [General] [answered]
`git-jump` (from git's contrib folder: https://github.com/git/git/tree/master/contrib/git-jump) is a shell command that open the results of `git jump diff` `git jump grep foo` `git jump merge` in your $EDITOR (in vim in a quickfix window).

I find it incredibly useful, and wondering if rg could use something similar. To wrap up, with `jump` we are telling rgc to open your editor and populate it with the results (instead of calling rg from the editor).

**Comment by BurntSushi** [comment, marked as answer]
I don't see a reason why this needs to be baked into ripgrep. I think it would be better to write a wrapper program for this.

**Comment by ltrzesniewski** [comment]
It's not exactly the same thing, but you may be interested in #2483. 🙂
